### ISSUE TITLE
Wrong order of no_blank_lines_after_phpdoc and no_whitespace_in_blank_line 

### ISSUE THREAD

**Issue by kubawerlos** [FriendsOfPHP/PHP-CS-Fixer#3369]
Tested on `master` by adding `tests/Fixtures/Integration/priority/no_blank_lines_after_phpdoc,no_whitespace_in_blank_line.test` with content:

```
--TEST--
Integration of fixers: no_blank_lines_after_phpdoc,no_whitespace_in_blank_line.
--RULESET--
{"no_blank_lines_after_phpdoc": true, "no_whitespace_in_blank_line": true}
--EXPECT--
<?php
namespace Foo;
/**
 * @package Bar
 */
class Baz
{
}

--INPUT--
<?php
namespace Foo;
/**
 * @package Bar
 */
 
class Baz
{
}

```
Notice single space in `INPUT` in line after PHPDoc.

**Comment by SpacePossum** [comment]
So IIUC `no_whitespace_in_blank_line` must go before `no_blank_lines_after_phpdoc`
Both fixers exist on 2.2 so I would expect the prio issue there as well?

**Comment by kubawerlos** [comment]
Yeah, fix will be for 2.2

and yes, `no_whitespace_in_blank_line` must go before `no_blank_lines_after_phpdoc` - there problem is with their current priorities - quite far away from each other: `no_whitespace_in_blank_line` has priority -19 and `no_blank_lines_after_phpdoc` 1

**Comment by julienfalque** [comment]
Fixed in #3466.
